### PR TITLE
rename built in node 

### PR DESCRIPTION
**Pull request by joelee2012**

### CODE REVIEW

File: api4jenkins/node.py [hunk 1]
```diff
@@ -106,7 +106,7 @@ def __iter__(self):
 
 class MasterComputer(Node):
     def __init__(self, jenkins, url):
-        super().__init__(jenkins, re.sub(r'/master/$', '/(master)/', url))
```
Comment (baydrift) [inline api4jenkins/node.py:109]: You will need both replacements in order to support both old and new versions of Jenkins.
'master' will only turn up on old, and 'Built-In Node' on new versions
Thank you for working on this

Comment (joelee2012) [reply api4jenkins/node.py:109]: changed, check again

File: api4jenkins/node.py [hunk 2]
```diff
@@ -106,7 +106,9 @@ def __iter__(self):
 
 class MasterComputer(Node):
     def __init__(self, jenkins, url):
-        super().__init__(jenkins, re.sub(r'/Built-In Node/$', '/(built-in)/', url))
+        # rename built-in node: https://www.jenkins.io/doc/upgrade-guide/2.319/
+        name = 'master' if url.endswith('/master/') else 'built-in'
```
Comment (baydrift) [inline api4jenkins/node.py:110]: I have been testing this, as I have two different versions of Jenkins on different urls on my network.
`Fatal Python error: Cannot recover from stack overflow.`
I suspect that it is recursing, to evaluate 'url'

File: api4jenkins/node.py [hunk 3]
```diff
@@ -106,7 +106,9 @@ def __iter__(self):
...
+        super().__init__(jenkins, f'{jenkins.url}computer/({name})/')
```
Comment (baydrift) [inline api4jenkins/node.py:111]: I have been testing this, as I have two different versions of Jenkins on different urls on my network.
I can confirm that it works. Thanks

**Review by baydrift — APPROVED** [review]

**Pull request merged**